### ISSUE TITLE
No option to choose installation location

### ISSUE THREAD

**Issue by Madelnick** [cheat-engine/cheat-engine#1755]
Hiya,

Just downloaded version 7.2. After two splashwindows to advertise for products, the installation process continues with the intallation in the system partition ("C:\Program Files\Cheat Engine 7.2\). Is this intended behavior?

**Comment by cheat-engine** [comment]
Yes. The current installer does not support customized installation destination "to make the install process more streamlined" (yeah)

**Comment by Madelnick** [comment]
LOLwut?

**Comment by Synthlight** [comment]
Fuck streamlined. I want it to be installed where I want it to be installed. I shouldn't have to install it, copy the files to where **I** want it, then uninstall, to get it there.

I'd have used the "if you wish to have the raw files without any configuration of file access permission by the setup click here" option too, except it still links to 7.2 so I can't.

I apologize if this is terse/rude. I just discovered this 'change' and am rather annoyed.

Please add the option back.

**Comment by Wryyyong** [comment]
> 
> 
> Yes. The current installer does not support customized installation destination "to make the install process more streamlined" (yeah)

Not sure who the hell informed you that would be a good idea. Something like an Express/Custom option during the installation would fare _much_ better, with Custom allowing the user to fine-tune their installation and Express handling everything for them.

**Comment by Madelnick** [comment]
> 
> 
> > Yes. The current installer does not support customized installation destination "to make the install process more streamlined" (yeah)
> 
> Not sure who the hell informed you that would be a good idea. Something like an Express/Custom option during the installation would fare _much_ better, with Custom allowing the user to fine-tune their installation and Express handling everything for them.

Dude it's the adware requiring them to implement it this way. I don't know exactly the mechanisms behind it. But I notice other apps with the same adware have the same "problem". It has to do with them choosing to commercialize clipgrab this way.

**Comment by avboy1337** [comment]
自己编译下OK的，每次自己编译的，win下。portable版本

**Comment by Madelnick** [comment]
@avboy1337
Most of us are simple end-users.

**Comment by cheat-engine** [comment]
fixed in the patreon version
